jc/parsers/efibootmgr.py

```diff
@@ -18,22 +18,72 @@
 
 Schema:
 
-    [
-      "BootCurrent": "0000",
-      "Timeout": "0 seconds",
-      "BootOrder": ["0000","0001"],
-      "BootOptions": [
-        {
-          "BootOptionReference": "Boot0000",
-          "DisplayName": "opensuse-secureboot",
-          "UefiDevicePath": "HD(1,GPT,e45dbd72-ba93-4810-9330-2c1c493ff08b,0x800,0x31800)/File(\EFI\opensuse\shim.efi)",
-          "BootOptionEnabled": true
-        }
-      ]
-    ]
+[
+    {
+        "BootCurrent": string,
+        "Timeout": string,
+        "BootOrder": [
+            string,
+            string,
+            string
+        ],
+        "BootOptions": [
+            {
+                "BootOptionReference": string,
+                "DisplayName": string,
+                "UefiDevicePath": string,
+                "BootOptionEnabled": bool
+            },
+            {
+                "BootOptionReference": string,
+                "DisplayName": string,
+                "UefiDevicePath": string,
+                "BootOptionEnabled": bool
+            },
+            {
+                "BootOptionReference": string,
+                "DisplayName": string,
+                "UefiDevicePath": string,
+                "BootOptionEnabled": bool
+            }
+        ]
+    }
+]
 
 Examples:
 
+$ sudo efibootmgr -v | jc --efibootmgr | jq
```

Comment:
this is also possible:
```suggestion
$ sudo efibootmgr -v | jc --efibootmgr --pretty
```